doc/sphinx-guides/source/admin/big-data-administration.rst

```diff
@@ -0,0 +1,312 @@
+Scaling Dataverse with Data Size
+================================
+
+This guide is intended to help administrators configure Dataverse appropriately to handle the amount of data their installation needs to manage.
+
+Scaling is a complex subject: there are many options available in Dataverse that can improve performance with larger scale data, some of which
+work differently than Dataverse's default configuration, potentially requiring user education, and some which can require additional expertise to manage.
+
+In general, there are three dimensions in which Dataverse can scale:
+  
+1. **Storage size of individual files and aggregate storage size**
+2. **Number of Files per Dataset**
+3. **Number of Datasets**
+
+
+.. contents:: |toctitle|
+        :local:
+
+Storage: Choosing the Right Store
+---------------------------------
+
+The main issues for handling larger files and larger aggregate data size relate to the performance of the storage used and how involved the Dataverse server is in the data transfer. 
+With appropriate configuration, Dataverse can support file sizes and aggregate dataset sizes into the terabyte scale and beyond.
+
+The primary choice in Dataverse related to storage is which type(s) of "store" (also called "Storage driver") to use:
+
+File Stores
+~~~~~~~~~~~
+
+The default storage option in Dataverse is based on a local file system. When files are transferred to Dataverse, they are first stored in a
+temporary location on the Dataverse server. Any zip files uploaded are unzipped to create multiple individual file entries. Once an upload is completed, 
+Dataverse copies the files to permanent storage. Dataverse also takes advantage of the file being local to inspect it's bytes to determine it's 
+MIME type, and, for tabular data, to 'ingest' it - extracting metadata about the variables used in the file and creating a tab-separated value (TSV)
+version of the file.
+
+Benefits: 
+
+- This option requires no external services and can potentially handle files into the gigabyte (GB) size range. For smaller institutions,
+  and in disciplines where datasets do not have more than a few hundred files and files are not too large, this can be the simplest option.
+- Unzipping of zip archives can be simpler for users than having to upload many individual files and was, at one time, the only way to 
+  preserve file path names when uploading.
```

Comment:
```suggestion
- Unzipping of zip archives can be simpler for users than having to upload many individual files and was, at one time, the only way to 
  preserve file path names when uploading. In addition, some of the unzipped files might be in a format that can be previewed or otherwise acted upon.
```

We could link to the file handling docs: https://guides.dataverse.org/en/6.8/user/dataset-management.html#file-handling